ThirdParty/Bolts/BoltsInClasspath/BoltsInClasspath/ViewController.m

```diff
@@ -111,54 +237,27 @@ -(void)forecastAtAppleButtonTapped {
 }
 
 -(void)allCornerCitiesForecastButtonTapped {
-    // executes all tasks in one BFTask
-    NSLog(@"Batch processing kicked off for -------------------------------multiple cities on Mega task executor executing each task concurrently.");
+    [self _printOutput:(@"Batch processing kicked off for -------------------------------multiple cities on Mega task executor executing each task concurrently.")];
```

Comment:
@andrejo-msft This is to let the user know the start of the process in the sample app console rather than in an IDE. so, I thought it would be nice for the user to see  whats going on even if he just installed the app on a mobile device